scripts/daqsystemtest_integtest_bundle.sh

```diff
@@ -93,6 +99,14 @@ while [[ ${overall_loop_count} -lt ${overall_run_count} ]]; do
         fi
 
         let individual_loop_count=${individual_loop_count}+1
+
+        if [[ ${stop_on_failure} -gt 0 ]]; then
+            search_result=`tail -20 ${ITGRUNNER_LOG_FILE} | grep -i fail`
```

Comment:
This seems slightly delicate. I wonder if the pytest return code could also be used.